bucket/buck.json

```diff
@@ -0,0 +1,34 @@
+{
+    "homepage": "http://buck.build/",
+    "version": "2019.05.22.01",
+    "description": "A high-performance build tool.",
+    "license": "Apache-2.0",
+    "url": "https://packages.chocolatey.org/buck.2019.05.22.01.nupkg",
+    "hash": "060b94b3a06581127ff1db41b2987eca52a6392d3e83cda1c3ec1b7c8b582390",
+    "extract_dir": "tools",
+    "depends": "versions/python27",
+    "suggest": {
+        "JDK8": [
+            "java/adopt8-hotspot",
+            "java/adopt8-openj9",
+            "java/corretto8",
+            "java/liberica8",
+            "java/ojdkbuild8",
+            "java/zulu8",
+            "java/zulufx8"
+        ]
+    },
+    "bin": "buck.bat",
+    "post_install": "(Get-Content \"$dir\\buck.bat\") -replace 'C:\\\\Python27', \"$(versiondir 'python27' 'current')\" | Set-Content \"$dir\\buck.bat\"",
+    "checkver": {
+        "url": "https://chocolatey.org/packages/buck",
```

Comment:
Why not go with github checkver?? https://github.com/facebook/buck/releases/latest

Reply:
Download URL is on `chocolatey.org`, so use choco checkver. `choco` checkver and hash PR hasn't been merged yet...